test/DateTest.php

```diff
@@ -118,6 +119,12 @@ public function testGetMessages()
         $this->assertEquals([], $this->validator->getMessages());
     }
 
+    public function testFormatDayTwoDigits()
+    {
+        $validator = new Date(['format' => 'd']);
+        $this->assertFalse($validator->isValid("6"));
```

Comment:
@gianarb I know but is better if PHPUnit fail with a better description message rather true is not equal to false :)